### PR TITLE
Add more tooltips

### PR DESCRIPTION
**Pull request by Imarok**

### CODE REVIEW

File: src/editor/editor.cpp [hunk 1]
```diff
@@ -546,6 +546,10 @@ void Editor::init_header_bar()
     attach_action_button(m_win.get_save_button(), ActionID::SAVE);
     attach_action_button(m_win.get_save_as_button(), ActionID::SAVE_AS);
 
+    m_win.get_new_button().set_tooltip_text("Create new document");
+    m_win.get_save_button().set_tooltip_text("Save");
```
Comment (carrotIndustries) [inline src/editor/editor.cpp:550]: That's redundant since the button is already labeled "Save".

File: src/editor/editor.cpp [hunk 2]
```diff
@@ -546,6 +546,10 @@ void Editor::init_header_bar()
...
+    m_win.get_save_as_button().set_tooltip_text("Save As");
```
Comment (carrotIndustries) [inline src/editor/editor.cpp:551]: Edit this in window.cmb using cambalanche and regenerate window.ui.

File: src/editor/editor.cpp [hunk 3]
```diff
@@ -546,6 +546,10 @@ void Editor::init_header_bar()
     attach_action_button(m_win.get_save_button(), ActionID::SAVE);
     attach_action_button(m_win.get_save_as_button(), ActionID::SAVE_AS);
 
+    m_win.get_new_button().set_tooltip_text("Create new document");
```
Comment (carrotIndustries) [inline src/editor/editor.cpp:549]: Edit this in window.cmb using cambalanche and regenerate window.ui.

**Comment by Imarok** [comment]
Seems like my version of Cambalache is newer than yours, so I just committed the immediate changes and not the changes due to a newer version of cambalache.

**Pull request merged**